### DISCUSSION TITLE
Not working!

### DISCUSSION THREAD

**Discussion by vncmenezes** [flamewave000/discord-overlay#34] [Q&A]
Hello! I followed every step in the guide, but the overlay does not work. Have installed net framework, have disabled and enabled everything, quit and enter call. Also have run discord overlay host as admin, but nothing happens! Any help?

**Comment by flamewave000** [comment]
Unfortunately Discord's overlay system can be finicky. Sometimes it can even take up to 4-5min for it to show up. If you followed all the steps exactly, it **should** work.
